<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FourPanel</class>
 <widget class="QMainWindow" name="FourPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QVTKWidget" name="view1"/>
      </item>
      <item row="0" column="1">
       <widget class="QVTKWidget" name="view2"/>
      </item>
      <item row="1" column="0">
       <widget class="QVTKWidget" name="view3"/>
      </item>
      <item row="1" column="1">
       <widget class="QVTKWidget" name="view4"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperate">
    <property name="title">
     <string>Operate</string>
    </property>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menuOperate"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset">
   <property name="text">
    <string>reset</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
